src/internal/m365/service/groups/backup.go

```diff
@@ -66,6 +67,15 @@ func ProduceBackupCollections(
 			// default one.
 			resp, err := ac.Groups().GetRootSite(ctx, bpc.ProtectedResource.ID())
 			if err != nil {
+				if clues.HasLabel(err, graph.LabelsNoProvisionResource) {
+					logger.Ctx(ctx).With("group_id", bpc.ProtectedResource.ID()).Infof("No SharePoint site found for group")
+
+					// TODO(meain): how should we handle a subservice missing?
```

Comment:
What exactly should we do here?

Reply:
Continue is fine.  A switch or set of conditions will be needed in the future, since different groups may have access to multiple different systems.  But the current design process you're working towards (error now, continue later, more after that) is a great approach.

Reply:
Cool, I've just updated the comment for now.